includes/api/class-wc-rest-order-notes-controller.php

```diff
@@ -374,7 +374,7 @@ public function get_item_schema() {
 		$schema = array(
 			'$schema'    => 'http://json-schema.org/draft-04/schema#',
 			'title'      => 'tax',
-			'type'       => 'order_note',
+			'type'       => 'object',
```

Comment:
`title` actually should have been order_note, not tax -- so that still needs fixed. Looks like a bad original copy & paste or something.